src/components/editor/Code.js

```diff
@@ -40,8 +43,13 @@ export class Codes extends Component<CodesProps> {
     return (
       <div>
         {this.props.codes.map((code, i) => {
-          return <Code key={i} onChange={this.props.onChange(i)} code={code} />
+          return <div>
```

Comment:
When I run `yarn test`, a warning is raised here because there is no key on this element.